server/routes/message.router.js

```diff
@@ -0,0 +1,124 @@
+const express = require('express');
+const pool = require('../modules/pool');
+const router = express.Router();
+const {
+  rejectUnauthenticated,
+} = require('../modules/authentication-middleware');
+
+router.get('/all', rejectUnauthenticated, (req, res) => {
```

Comment:
REST API convention would be to call this just `.get('/')`. It's implied by REST conventions that that means "get all"